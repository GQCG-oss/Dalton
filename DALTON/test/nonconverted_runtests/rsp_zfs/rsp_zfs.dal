**DALTON
.RUN RESPONSE
**WAVE FUNCTION
.HF
*SCF INPUT
.DOUBLY
 3 1 1 0 2 0 0 0
.SINGLY
 0 0 0 0 0 1 1 0
**RESPONSE
*ESR
.ZFS
**END OF
