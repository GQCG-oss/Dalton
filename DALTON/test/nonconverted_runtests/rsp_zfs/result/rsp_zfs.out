

     ************************************************************************
     *************** Dalton - An Electronic Structure Program ***************
     ************************************************************************

    This is output from DALTON (Release Dalton2013 patch 0)
   ----------------------------------------------------------------------------
    NOTE:
     
    Dalton is an experimental code for the evaluation of molecular
    properties using (MC)SCF, DFT, CI, and CC wave functions.
    The authors accept no responsibility for the performance of
    the code or for the correctness of the results.
     
    The code (in whole or part) is provided under a licence and
    is not to be reproduced for further distribution without
    the written permission of the authors or their representatives.
     
    See the home page "http://daltonprogram.org" for further information.
     
    If results obtained with this code are published,
    the appropriate citations would be both of:
     
       K. Aidas, C. Angeli, K. L. Bak, V. Bakken, R. Bast,
       L. Boman, O. Christiansen, R. Cimiraglia, S. Coriani,
       P. Dahle, E. K. Dalskov, U. Ekstroem, T. Enevoldsen,
       J. J. Eriksen, P. Ettenhuber, B. Fernandez, L. Ferrighi,
       H. Fliegl, L. Frediani, K. Hald, A. Halkier, C. Haettig,
       H. Heiberg, T. Helgaker, A. C. Hennum, H. Hettema,
       E. Hjertenaes, S. Hoest, I.-M. Hoeyvik, M. F. Iozzi,
       B. Jansik, H. J. Aa. Jensen, D. Jonsson, P. Joergensen,
       J. Kauczor, S. Kirpekar, T. Kjaergaard, W. Klopper,
       S. Knecht, R. Kobayashi, H. Koch, J. Kongsted, A. Krapp,
       K. Kristensen, A. Ligabue, O. B. Lutnaes, J. I. Melo,
       K. V. Mikkelsen, R. H. Myhre, C. Neiss, C. B. Nielsen,
       P. Norman, J. Olsen, J. M. H. Olsen, A. Osted,
       M. J. Packer, F. Pawlowski, T. B. Pedersen, P. F. Provasi,
       S. Reine, Z. Rinkevicius, T. A. Ruden, K. Ruud, V. Rybkin,
       P. Salek, C. C. M. Samson, A. Sanchez de Meras, T. Saue,
       S. P. A. Sauer, B. Schimmelpfennig, K. Sneskov,
       A. H. Steindal, K. O. Sylvester-Hvid, P. R. Taylor,
       A. M. Teale, E. I. Tellgren, D. P. Tew, A. J. Thorvaldsen,
       L. Thoegersen, O. Vahtras, M. A. Watson, D. J. D. Wilson,
       M. Ziolkowski and H. Agren.
       The Dalton quantum chemistry program system.
       WIREs Comput. Mol. Sci. 2013. doi: 10.1002/wcms.1172
    
    and
    
       Dalton, a Molecular Electronic Structure Program,
       Release DALTON2013.0 (2013), see http://daltonprogram.org
   ----------------------------------------------------------------------------

    Authors in alphabetical order (major contribution(s) in parenthesis):

  Kestutis Aidas,           Vilnius University,           Lithuania   (QM/MM)
  Celestino Angeli,         University of Ferrara,        Italy       (NEVPT2)
  Keld L. Bak,              UNI-C,                        Denmark     (AOSOPPA, non-adiabatic coupling, magnetic properties)
  Vebjoern Bakken,          University of Oslo,           Norway      (DALTON; geometry optimizer, symmetry detection)
  Radovan Bast,             KTH Stockholm                 Sweden      (DALTON installation and execution frameworks)
  Linus Boman,              NTNU,                         Norway      (Cholesky decomposition and subsystems)
  Ove Christiansen,         Aarhus University,            Denmark     (CC module)
  Renzo Cimiraglia,         University of Ferrara,        Italy       (NEVPT2)
  Sonia Coriani,            University of Trieste,        Italy       (CC module, MCD in RESPONS)
  Paal Dahle,               University of Oslo,           Norway      (Parallelization)
  Erik K. Dalskov,          UNI-C,                        Denmark     (SOPPA)
  Thomas Enevoldsen,        Univ. of Southern Denmark,    Denmark     (SOPPA)
  Janus J. Eriksen,         Aarhus University,            Denmark     (PE-MP2/SOPPA, TDA)
  Berta Fernandez,          U. of Santiago de Compostela, Spain       (doublet spin, ESR in RESPONS)
  Lara Ferrighi,            Aarhus University,            Denmark     (PCM Cubic response)
  Heike Fliegl,             University of Oslo,           Norway      (CCSD(R12))
  Luca Frediani,            UiT The Arctic U. of Norway,  Norway      (PCM)
  Bin Gao,                  UiT The Arctic U. of Norway,  Norway      (Gen1Int library)
  Christof Haettig,         Ruhr-University Bochum,       Germany     (CC module)
  Kasper Hald,              Aarhus University,            Denmark     (CC module)
  Asger Halkier,            Aarhus University,            Denmark     (CC module)
  Hanne Heiberg,            University of Oslo,           Norway      (geometry analysis, selected one-electron integrals)
  Trygve Helgaker,          University of Oslo,           Norway      (DALTON; ABACUS, ERI, DFT modules, London, and much more)
  Alf Christian Hennum,     University of Oslo,           Norway      (Parity violation)
  Hinne Hettema,            University of Auckland,       New Zealand (quadratic response in RESPONS; SIRIUS supersymmetry)
  Eirik Hjertenaes,         NTNU,                         Norway      (Cholesky decomposition)
  Maria Francesca Iozzi,    University of Oslo,           Norway      (RPA)
  Brano Jansik              Technical Univ. of Ostrava    Czech Rep.  (DFT cubic response)
  Hans Joergen Aa. Jensen,  Univ. of Southern Denmark,    Denmark     (DALTON; SIRIUS, RESPONS, ABACUS modules, London, and much more)
  Dan Jonsson,              UiT The Arctic U. of Norway,  Norway      (cubic response in RESPONS module)
  Poul Joergensen,          Aarhus University,            Denmark     (RESPONS, ABACUS, and CC modules)
  Joanna Kauczor,           Linkoeping University,        Sweden      (Complex polarization propagator (CPP) module)
  Sheela Kirpekar,          Univ. of Southern Denmark,    Denmark     (Mass-velocity & Darwin integrals)
  Wim Klopper,              KIT Karlsruhe,                Germany     (R12 code in CC, SIRIUS, and ABACUS modules)
  Stefan Knecht,            ETH Zurich,                   Switzerland (Parallel CI and MCSCF)
  Rika Kobayashi,           Australian National Univ.,    Australia   (DIIS in CC, London in MCSCF)
  Henrik Koch,              NTNU,                         Norway      (CC module, Cholesky decomposition)
  Jacob Kongsted,           Univ. of Southern Denmark,    Denmark     (Polarizable embedding, QM/MM)
  Andrea Ligabue,           University of Modena,         Italy       (CTOCD, AOSOPPA)
  Ola B. Lutnaes,           University of Oslo,           Norway      (DFT Hessian)
  Juan I. Melo,             University of Buenos Aires,   Argentina   (LRESC, Relativistic Effects on NMR Shieldings)
  Kurt V. Mikkelsen,        University of Copenhagen,     Denmark     (MC-SCRF and QM/MM)
  Rolf H. Myhre,            NTNU,                         Norway      (Cholesky, subsystems and ECC2)
  Christian Neiss,          Univ. Erlangen-Nuernberg,     Germany     (CCSD(R12))
  Christian B. Nielsen,     University of Copenhagen,     Denmark     (QM/MM)
  Patrick Norman,           Linkoeping University,        Sweden      (Cubic response and complex response in RESPONS)
  Jeppe Olsen,              Aarhus University,            Denmark     (SIRIUS CI/density modules)
  Jogvan Magnus H. Olsen,   Univ. of Southern Denmark,    Denmark     (Polarizable embedding, PE library, QM/MM)
  Anders Osted,             Copenhagen University,        Denmark     (QM/MM)
  Martin J. Packer,         University of Sheffield,      UK          (SOPPA)
  Filip Pawlowski,          Kazimierz Wielki University   Poland      (CC3)
  Thomas B. Pedersen,       University of Oslo,           Norway      (Cholesky decomposition)
  Patricio F. Provasi,      University of Northeastern,   Argentina   (Analysis of coupling constants in localized orbitals)
  Zilvinas Rinkevicius,     KTH Stockholm,                Sweden      (open-shell DFT, ESR)
  Elias Rudberg,            KTH Stockholm,                Sweden      (DFT grid and basis info)
  Torgeir A. Ruden,         University of Oslo,           Norway      (Numerical derivatives in ABACUS)
  Kenneth Ruud,             UiT The Arctic U. of Norway,  Norway      (DALTON; ABACUS magnetic properties and  much more)
  Pawel Salek,              KTH Stockholm,                Sweden      (DALTON; DFT code)
  Claire C. M. Samson       University of Karlsruhe       Germany     (Boys localization, r12 integrals in ERI)
  Alfredo Sanchez de Meras, University of Valencia,       Spain       (CC module, Cholesky decomposition)
  Trond Saue,               Paul Sabatier University,     France      (direct Fock matrix construction)
  Stephan P. A. Sauer,      University of Copenhagen,     Denmark     (SOPPA(CCSD), SOPPA prop., AOSOPPA, vibrational g-factors)
  Bernd Schimmelpfennig,    Forschungszentrum Karlsruhe,  Germany     (AMFI module)
  Kristian Sneskov,         Aarhus University,            Denmark     (QM/MM, PE-CC)
  Arnfinn H. Steindal,      UiT The Arctic U. of Norway,  Norway      (parallel QM/MM)
  K. O. Sylvester-Hvid,     University of Copenhagen,     Denmark     (MC-SCRF)
  Peter R. Taylor,          VLSCI/Univ. of Melbourne,     Australia   (Symmetry handling ABACUS, integral transformation)
  Andrew M. Teale,          University of Nottingham,     England     (DFT-AC, DFT-D)
  David P. Tew,             University of Bristol,        England     (CCSD(R12))
  Olav Vahtras,             KTH Stockholm,                Sweden      (triplet response, spin-orbit, ESR, TDDFT, open-shell DFT)
  David J. Wilson,          La Trobe University,          Australia   (DFT Hessian and DFT magnetizabilities)
  Hans Agren,               KTH Stockholm,                Sweden      (SIRIUS module, RESPONS, MC-SCRF solvation model)
 --------------------------------------------------------------------------------

     Date and time (Linux)  : Sun Sep  8 20:39:21 2013
     Host name              : lpqlx131.ups-tlse.fr                    

 * Work memory size             :    64000000 =  488.28 megabytes.

 * Directories for basis set searches:
   1) /home/bast/DALTON-2013.0-Source/build/test_rsp_zfs
   2) /home/bast/DALTON-2013.0-Source/build/basis


Compilation information
-----------------------

 Who compiled             | bast
 Host                     | lpqlx131.ups-tlse.fr
 System                   | Linux-3.8.5-201.fc18.x86_64
 CMake generator          | Unix Makefiles
 Processor                | x86_64
 64-bit integers          | OFF
 MPI                      | OFF
 Fortran compiler         | /usr/bin/gfortran
 Fortran compiler version | GNU Fortran (GCC) 4.7.2 20121109 (Red Hat 4.7.2-8)
 C compiler               | /usr/bin/gcc
 C compiler version       | gcc (GCC) 4.7.2 20121109 (Red Hat 4.7.2-8)
 C++ compiler             | /usr/bin/g++
 C++ compiler version     | g++ (GCC) 4.7.2 20121109 (Red Hat 4.7.2-8)
 Static linking           | OFF
 Last Git revision        | f34203295a86316e27f9e7b44f9b6769c4a046c0
 Configuration time       | 2013-09-08 20:31:27.952056


   Content of the .dal input file
 ----------------------------------

**DALTON                                          
.RUN RESPONSE                                     
**WAVE FUNCTION                                   
.HF                                               
*SCF INPUT                                        
.DOUBLY                                           
 3 1 1 0 2 0 0 0                                  
.SINGLY                                           
 0 0 0 0 0 1 1 0                                  
**RESPONSE                                        
*ESR                                              
.ZFS                                              
**END OF                                          


   Content of the .mol file
 ----------------------------

BASIS                                                                          
cc-pVDZ                                                                        
Test1 i test-suite: Geometrioptimering med symmetri, med beregning av          
egenskaper i foerste og siste punkt. Generelt kontraherte basissett            
    1                                                                          
        8.    2                                                                
O     0.00000000     0.00000000     1.1405                                     
O     0.00000000     0.00000000    -1.1405                                     


       *******************************************************************
       *********** Output from DALTON general input processing ***********
       *******************************************************************

 --------------------------------------------------------------------------------
   Overall default print level:    0
   Print level for DALTON.STAT:    1

    HERMIT 1- and 2-electron integral sections will be executed
    "Old" integral transformation used (limited to max 255 basis functions)
    Wave function sections will be executed (SIRIUS module)
    Dynamic molecular response properties section will be executed (RESPONSE module)
 --------------------------------------------------------------------------------


   ****************************************************************************
   *************** Output of molecule and basis set information ***************
   ****************************************************************************


    The two title cards from your ".mol" input:
    ------------------------------------------------------------------------
 1: Test1 i test-suite: Geometrioptimering med symmetri, med beregning av   
 2: egenskaper i foerste og siste punkt. Generelt kontraherte basissett     
    ------------------------------------------------------------------------

  Atomic type no.    1
  --------------------
  Nuclear charge:   8.00000
  Number of symmetry independent centers:    2
  Number of basis sets to read;    2
  Basis set file used for this atomic type with Z =   8 :
     "/home/bast/DALTON-2013.0-Source/build/basis/cc-pVDZ"


                      SYMADD: Requested addition of symmetry
                      --------------------------------------

 Symmetry test threshold:  5.00E-06

 - molecule centered at center of mass and rotated so
   principal axes of inertia are along coordinate axes.

 Symmetry class found: D(oo,h)        

 Symmetry Independent Centres             
 ----------------------------
       8 :      0.00000000     0.00000000     1.14050000  Isotope  1

 The following elements were found:   X  Y  Z  


                         SYMGRP: Point group information
                         -------------------------------

Full point group is: D(oo,h)        
Represented as:      D2h

   * The point group was generated by:

      Reflection in the yz-plane
      Reflection in the xz-plane
      Reflection in the xy-plane

   * Group multiplication table

        |  E   C2z  C2y  C2x   i   Oxy  Oxz  Oyz
   -----+----------------------------------------
     E  |  E   C2z  C2y  C2x   i   Oxy  Oxz  Oyz
    C2z | C2z   E   C2x  C2y  Oxy   i   Oyz  Oxz
    C2y | C2y  C2x   E   C2z  Oxz  Oyz   i   Oxy
    C2x | C2x  C2y  C2z   E   Oyz  Oxz  Oxy   i 
     i  |  i   Oxy  Oxz  Oyz   E   C2z  C2y  C2x
    Oxy | Oxy   i   Oyz  Oxz  C2z   E   C2x  C2y
    Oxz | Oxz  Oyz   i   Oxy  C2y  C2x   E   C2z
    Oyz | Oyz  Oxz  Oxy   i   C2x  C2y  C2z   E 

   * Character table

        |  E   C2z  C2y  C2x   i   Oxy  Oxz  Oyz
   -----+----------------------------------------
    Ag  |   1    1    1    1    1    1    1    1
    B3u |   1   -1   -1    1   -1    1    1   -1
    B2u |   1   -1    1   -1   -1    1   -1    1
    B1g |   1    1   -1   -1    1    1   -1   -1
    B1u |   1    1   -1   -1   -1   -1    1    1
    B2g |   1   -1    1   -1    1   -1    1   -1
    B3g |   1   -1   -1    1    1   -1   -1    1
    Au  |   1    1    1    1   -1   -1   -1   -1

   * Direct product table

        | Ag   B3u  B2u  B1g  B1u  B2g  B3g  Au 
   -----+----------------------------------------
    Ag  | Ag   B3u  B2u  B1g  B1u  B2g  B3g  Au 
    B3u | B3u  Ag   B1g  B2u  B2g  B1u  Au   B3g
    B2u | B2u  B1g  Ag   B3u  B3g  Au   B1u  B2g
    B1g | B1g  B2u  B3u  Ag   Au   B3g  B2g  B1u
    B1u | B1u  B2g  B3g  Au   Ag   B3u  B2u  B1g
    B2g | B2g  B1u  Au   B3g  B3u  Ag   B1g  B2u
    B3g | B3g  Au   B1u  B2g  B2u  B1g  Ag   B3u
    Au  | Au   B3g  B2g  B1u  B1g  B2u  B3u  Ag 


                                 Isotopic Masses
                                 ---------------

                           O   _1     15.994915
                           O   _2     15.994915

                       Total mass:    31.989830 amu
                       Natural abundance:  99.521 %

 Center-of-mass coordinates (a.u.):    0.000000    0.000000    0.000000


  Atoms and basis sets
  --------------------

  Number of atom types :    1
  Total number of atoms:    2

  Basis set used is "cc-pVDZ" from the basis set library.

  label    atoms   charge   prim   cont     basis
  ----------------------------------------------------------------------
  O           2    8.0000    26    14      [9s4p1d|3s2p1d]                                    
  ----------------------------------------------------------------------
  total:      2   16.0000    52    28
  ----------------------------------------------------------------------
  Spherical harmonic basis used.

  Threshold for neglecting AO integrals:  1.00D-12


  Cartesian Coordinates (a.u.)
  ----------------------------

  Total number of coordinates:    6
  O   / 1 :     1  x   0.0000000000    2  y   0.0000000000    3  z   1.1405000000
  O   / 2 :     4  x   0.0000000000    5  y   0.0000000000    6  z  -1.1405000000


  Symmetry Coordinates
  --------------------

  Number of coordinates in each symmetry:     1    1    1    0    1    1    1    0

  Symmetry  Ag  ( 1)

    1   O     z    [  3  -    6 ]/2

  Symmetry  B3u ( 2)

    2   O     x    [  1  +    4 ]/2

  Symmetry  B2u ( 3)

    3   O     y    [  2  +    5 ]/2

  Symmetry  B1u ( 5)

    4   O     z    [  3  +    6 ]/2

  Symmetry  B2g ( 6)

    5   O     x    [  1  -    4 ]/2

  Symmetry  B3g ( 7)

    6   O     y    [  2  -    5 ]/2


   Interatomic separations (in Angstrom):
   --------------------------------------

            O   _1      O   _2
            ------      ------
 O   _1:    0.000000
 O   _2:    1.207053    0.000000


  Max    interatomic separation is    1.2071 Angstrom (    2.2810 Bohr)
  between atoms    2 and    1, "O   _2" and "O   _1".

  Min YX interatomic separation is    1.2071 Angstrom (    2.2810 Bohr)


  Bond distances (Angstrom):
  --------------------------

                  atom 1     atom 2       distance
                  ------     ------       --------
  bond distance:  O   _2     O   _1       1.207053




 Principal moments of inertia (u*A**2) and principal axes
 --------------------------------------------------------

   IA       0.000000          0.000000    0.000000    1.000000
   IB      11.652115          0.000000    1.000000    0.000000
   IC      11.652115          1.000000    0.000000    0.000000


 Rotational constants
 --------------------

 The molecule is linear.

               B =        43372.30 MHz     (    1.446744 cm-1)


@  Nuclear repulsion energy :   28.057869355546 Hartree


  Symmetry Orbitals
  -----------------

  Number of orbitals in each symmetry:           7    3    3    1    7    3    3    1


  Symmetry  Ag ( 1)

    1     O        1s         1 +    2
    2     O        1s         3 +    4
    3     O        1s         5 +    6
    4     O        2pz       11 -   12
    5     O        2pz       17 -   18
    6     O        3d0       23 +   24
    7     O        3d2+      27 +   28


  Symmetry  B3u( 2)

    8     O        2px        7 +    8
    9     O        2px       13 +   14
   10     O        3d1+      25 -   26


  Symmetry  B2u( 3)

   11     O        2py        9 +   10
   12     O        2py       15 +   16
   13     O        3d1-      21 -   22


  Symmetry  B1g( 4)

   14     O        3d2-      19 +   20


  Symmetry  B1u( 5)

   15     O        1s         1 -    2
   16     O        1s         3 -    4
   17     O        1s         5 -    6
   18     O        2pz       11 +   12
   19     O        2pz       17 +   18
   20     O        3d0       23 -   24
   21     O        3d2+      27 -   28


  Symmetry  B2g( 6)

   22     O        2px        7 -    8
   23     O        2px       13 -   14
   24     O        3d1+      25 +   26


  Symmetry  B3g( 7)

   25     O        2py        9 -   10
   26     O        2py       15 -   16
   27     O        3d1-      21 +   22


  Symmetry  Au ( 8)

   28     O        3d2-      19 -   20

  Symmetries of electric field:  B3u(2)  B2u(3)  B1u(5)

  Symmetries of magnetic field:  B3g(7)  B2g(6)  B1g(4)


                     .---------------------------------------.
                     | Starting in Integral Section (HERMIT) |
                     `---------------------------------------'



    *************************************************************************
    ****************** Output from HERMIT input processing ******************
    *************************************************************************



     ************************************************************************
     ************************** Output from HERINT **************************
     ************************************************************************


 Threshold for neglecting two-electron integrals:  1.00D-12
 Number of two-electron integrals written:       10381 ( 12.6% )
 Megabytes written:                              0.124

 >>>> Total CPU  time used in HERMIT:   0.03 seconds
 >>>> Total wall time used in HERMIT:   0.03 seconds


                        .----------------------------------.
                        | End of Integral Section (HERMIT) |
                        `----------------------------------'



                   .--------------------------------------------.
                   | Starting in Wave Function Section (SIRIUS) |
                   `--------------------------------------------'


 *** Output from Huckel module :

     Using EWMO model:          F
     Using EHT  model:          T
     Number of Huckel orbitals each symmetry:    3    1    1    0    3    1    1    0

 Huckel EHT eigenvalues for symmetry :  1
          -20.692822      -1.610593      -0.632918

 Huckel EHT eigenvalues for symmetry :  2
           -0.728576

 Huckel EHT eigenvalues for symmetry :  3
           -0.728576

 Huckel EHT eigenvalues for symmetry :  5
          -20.693781      -1.090207      -0.374680

 Huckel EHT eigenvalues for symmetry :  6
           -0.503824

 Huckel EHT eigenvalues for symmetry :  7
           -0.503824

 **********************************************************************
 *SIRIUS* a direct, restricted step, second order MCSCF program       *
 **********************************************************************

 
     Date and time (Linux)  : Sun Sep  8 20:39:21 2013
     Host name              : lpqlx131.ups-tlse.fr                    

 Title lines from ".mol" input file:
     Test1 i test-suite: Geometrioptimering med symmetri, med beregning av   
     egenskaper i foerste og siste punkt. Generelt kontraherte basissett     

 Print level on unit LUPRI =   2 is   0
 Print level on unit LUW4  =   2 is   5

@    Restricted, high spin open shell Hartree-Fock calculation.

@    Time-dependent Hartree-Fock calculation (random phase approximation).

 Initial molecular orbitals are obtained according to
 ".MOSTART EHT   " input option

     Wave function specification
     ============================
@    For the wave function of type :      >>> HF <<<
@    Number of closed shell electrons          14
@    Number of electrons in active shells       2
@    Total charge of the molecule               0

@    Spin multiplicity and 2 M_S                3         2
     Total number of symmetries                 8
@    Reference state symmetry                   4

     Orbital specifications
     ======================
     Abelian symmetry species          All |    1    2    3    4    5    6    7    8
                                       --- |  ---  ---  ---  ---  ---  ---  ---  ---
@    Occupied SCF orbitals               7 |    3    1    1    0    2    0    0    0
@    Open shell SCF orbitals             2 |    0    0    0    0    0    1    1    0
     Secondary orbitals                 19 |    4    2    2    1    5    2    2    1
     Total number of orbitals           28 |    7    3    3    1    7    3    3    1
     Number of basis functions          28 |    7    3    3    1    7    3    3    1

     Optimization information
     ========================
@    Number of configurations                 1
@    Number of orbital rotations             30
     ------------------------------------------
@    Total number of variables               31

     Maximum number of Fock   iterations      0
     Maximum number of DIIS   iterations     60
     Maximum number of QC-SCF iterations     60
     Threshold for SCF convergence     1.00D-05


 >>>>> DIIS optimization of Hartree-Fock <<<<<

 C1-DIIS algorithm; max error vectors =   10

 Iter      Total energy        Error norm    Delta(E)  DIIS dim.
 -----------------------------------------------------------------------------
@  1    -149.390433642        2.48261D+00   -1.49D+02    1
      Virial theorem: -V/T =      2.006855
@      MULPOP O   _1  0.00; O   _2  0.00; 
 -----------------------------------------------------------------------------
@  2    -149.597146577        4.35539D-01   -2.07D-01    2
      Virial theorem: -V/T =      1.996415
@      MULPOP O   _1  0.00; O   _2  0.00; 
 -----------------------------------------------------------------------------
@  3    -149.607453292        1.12510D-01   -1.03D-02    3
      Virial theorem: -V/T =      2.002245
@      MULPOP O   _1 -0.00; O   _2 -0.00; 
 -----------------------------------------------------------------------------
@  4    -149.608139187        2.01724D-02   -6.86D-04    4
      Virial theorem: -V/T =      2.000690
@      MULPOP O   _1  0.00; O   _2  0.00; 
 -----------------------------------------------------------------------------
@  5    -149.608170752        2.78710D-03   -3.16D-05    5
      Virial theorem: -V/T =      2.000841
@      MULPOP O   _1  0.00; O   _2  0.00; 
 -----------------------------------------------------------------------------
@  6    -149.608171278        3.53808D-04   -5.25D-07    6
      Virial theorem: -V/T =      2.000826
@      MULPOP O   _1 -0.00; O   _2 -0.00; 
 -----------------------------------------------------------------------------
@  7    -149.608171287        3.62238D-05   -9.82D-09    7
      Virial theorem: -V/T =      2.000827
@      MULPOP O   _1  0.00; O   _2  0.00; 
 -----------------------------------------------------------------------------
@  8    -149.608171287        2.46903D-06   -1.10D-10    8

@ *** DIIS converged in   8 iterations !
@     Converged SCF energy, gradient:   -149.608171287437    2.47D-06
    - total time used in SIRFCK :              0.00 seconds


 *** SCF orbital energy analysis ***

 Orbital energy analysis for an open-shell system.
 Orbital energies are not well-defined for open-shell systems
   here is used block diagonalization of the FD=FC+FV Fock matrix.
 NOTE that Koopmans' theorem is not fulfilled for this case.

 Only the five lowest virtual orbital energies printed in each symmetry.

 Number of electrons :   14
 Orbital occupations :    3    1    1    0    2    0    0    0

 Sym       Hartree-Fock orbital energies

  1    -20.72765289    -1.65384718    -0.72955205     1.15388734     1.32900829
         2.68345291     3.18123478

  2     -0.70194210     1.11025642     2.40269858

  3     -0.70194210     1.11025642     2.40269858

  4      2.68345291

  5    -20.72672192    -1.09760345     0.46463202     1.07973386     1.97430560
         3.00950383     4.19270969

  6     -0.20702929     1.21686773     3.68024662

  7     -0.20702929     1.21686773     3.68024662

  8      3.00950383

    E(LUMO) :     0.46463202 au (symmetry 5)
  - E(HOMO) :    -0.70194210 au (symmetry 2)
  ------------------------------------------
    gap     :     1.16657412 au

and energy and symmetry of HSROHF open shell orbitals:
    -0.20702929         6
    -0.20702929         7

 NOTE: MOLECULAR ORBITALS ARE NOT CANONICAL HARTREE-FOCK ORBITALS

 Largest off-diagonal Fock matrix element is  6.96D-02

 >>> Writing SIRIFC interface file <<<

 >>>> CPU and wall time for SCF :       0.007       0.007


                       .-----------------------------------.
                       | >>> Final results from SIRIUS <<< |
                       `-----------------------------------'


@    Spin multiplicity:           3
@    Spatial symmetry:            4
@    Total charge of molecule:    0

@    Final HF energy:            -149.608171287437                 
@    Nuclear repulsion:            28.057869355546
@    Electronic energy:          -177.666040642983

@    Final gradient norm:           0.000002469026

 
     Date and time (Linux)  : Sun Sep  8 20:39:21 2013
     Host name              : lpqlx131.ups-tlse.fr                    

 (Only coefficients >0.0100 are printed.)

 Molecular orbitals for symmetry species  1
 ------------------------------------------

    Orbital         1        2        3        4        5
   1 O   :1s     0.7072   0.0004  -0.0091  -0.0126   0.4490
   2 O   :1s     0.0007   0.6600   0.1947  -0.1640   2.1345
   3 O   :1s    -0.0002  -0.1092   0.1536   0.3955  -2.1399
   4 O   :2pz   -0.0018  -0.2313   0.6771  -0.8574  -0.0057
   5 O   :2pz    0.0014   0.0641  -0.0692   1.2139   0.0746
   6 O   :3d0   -0.0003   0.0260  -0.0318   0.1024   0.1249

 Molecular orbitals for symmetry species  2
 ------------------------------------------

    Orbital         1        2        3
   1 O   :2px    0.6490  -0.9889   0.1429
   2 O   :2px   -0.0152   1.0788  -0.0381
   3 O   :3d1+  -0.0331   0.0440   0.6382

 Molecular orbitals for symmetry species  3
 ------------------------------------------

    Orbital         1        2        3
   1 O   :2py    0.6490  -0.9889   0.1429
   2 O   :2py   -0.0152   1.0788  -0.0381
   3 O   :3d1-  -0.0331   0.0440   0.6382

 Molecular orbitals for symmetry species  4
 ------------------------------------------

    Orbital         1
   1 O   :3d2-   0.6914

 Molecular orbitals for symmetry species  5
 ------------------------------------------

    Orbital         1        2        3        4
   1 O   :1s     0.7074   0.0049   0.0148  -0.0056
   2 O   :1s     0.0012   0.7116  -0.3644   0.0664
   3 O   :1s    -0.0027   0.0467  -1.0321   0.9475
   4 O   :2pz   -0.0018   0.2042   0.5961   1.0230
   5 O   :2pz    0.0023  -0.0408   0.9262  -1.8026
   6 O   :3d0    0.0004  -0.0104   0.0143   0.0936

 Molecular orbitals for symmetry species  6
 ------------------------------------------

    Orbital         1        2        3
   1 O   :2px    0.7774  -0.9231   0.0110
   2 O   :2px    0.0383   1.4848  -0.2807
   3 O   :3d1+   0.0050   0.0090   0.8367

 Molecular orbitals for symmetry species  7
 ------------------------------------------

    Orbital         1        2        3
   1 O   :2py    0.7774  -0.9231   0.0110
   2 O   :2py    0.0383   1.4848  -0.2807
   3 O   :3d1-   0.0050   0.0090   0.8367

 Molecular orbitals for symmetry species  8
 ------------------------------------------

    Orbital         1
   1 O   :3d2-   0.7239



 >>>> Total CPU  time used in SIRIUS :      0.01 seconds
 >>>> Total wall time used in SIRIUS :      0.01 seconds

 
     Date and time (Linux)  : Sun Sep  8 20:39:21 2013
     Host name              : lpqlx131.ups-tlse.fr                    


                     .---------------------------------------.
                     | End of Wave Function Section (SIRIUS) |
                     `---------------------------------------'



                 .------------------------------------------------.
                 | Starting in Dynamic Property Section (RESPONS) |
                 `------------------------------------------------'


 ------------------------------------------------------------------------------
  RESPONSE  -  an MCSCF, MC-srDFT, DFT, and SOPPA response property program
 ------------------------------------------------------------------------------


 <<<<<<<<<< OUTPUT FROM RESPONSE INPUT PROCESSING >>>>>>>>>>



 ********* ESRINP ********


 Changes of defaults under *ESR   :
 ----------------------------------


 ESR - zero-field splitting  calculation carried out


 high-spin open-shell SCF energy:     -149.608171287436761
 -- inactive part               :     -177.344705039705275
 --   active part               :       -0.321335603277318
 -- nuclear repulsion           :       28.057869355545812


                   *******************************************
                   *** High-spin ROHF response calculation ***
                   *******************************************



 Output from ZFSDRV
 ------------------



                        Two-electron spin-spin interaction
                        ----------------------------------


               Column   1     Column   2     Column   3
       1       0.04294899     0.00000000     0.00000000
       2       0.00000000     0.04294899     0.00000000
       3       0.00000000     0.00000000    -0.08589798
    ==== End of matrix output ====


 Trace-less zero field splitting tensor (cm-1)
 ---------------------------------------------


               Column   1     Column   2     Column   3
       1      -0.50312349    -0.00000000    -0.00000000
       2      -0.00000000    -0.50312349    -0.00000000
       3      -0.00000000    -0.00000000     1.00624698
    ==== End of matrix output ====


 ZFS energy eigenvalues (cm-1)
 -----------------------------


               Column   1
       1      -1.00624698
       2       0.50312349
       3       0.50312349
    ==== End of matrix output ====


 Zero field splitting paramters
 ------------------------------

@ZFS parameter D =   1.509370 cm-1 =   45249.79 MHz
@ZFS parameter E =   0.000000 cm-1 =       0.00 MHz


 ZFS tensor eigenvalues and cosines
 ----------------------------------

            cm-1             MHz       direction cosines
       -0.503123   -15083.262725  0.0000  1.0000  0.0000
       -0.503123   -15083.262725  1.0000  0.0000  0.0000
        1.006247    30166.525451  0.0000  0.0000  1.0000

 >>>> Total CPU  time used in RESPONSE:   0.08 seconds
 >>>> Total wall time used in RESPONSE:   0.08 seconds


                   .-------------------------------------------.
                   | End of Dynamic Property Section (RESPONS) |
                   `-------------------------------------------'

 >>>> Total CPU  time used in DALTON:   0.12 seconds
 >>>> Total wall time used in DALTON:   0.12 seconds

 
     Date and time (Linux)  : Sun Sep  8 20:39:21 2013
     Host name              : lpqlx131.ups-tlse.fr                    
